tests/test_family.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+from manim import *
```

Comment:
```suggestion
from manim import Mobject, VMobject, RIGHT
```

or 

```suggestion
from manim import Mobject, Circle, RIGHT
```

I think this'll help us narrow down any potential sources of problems instead of having the whole of `manim` be suspect.